src/test/java/io/strimzi/kafka/bridge/http/ConsumerGeneratedNameIT.java

```diff
@@ -50,14 +50,23 @@
 @DisabledIfEnvironmentVariable(named = "EXTERNAL_BRIDGE", matches = "((?i)TRUE(?-i))")
 public class ConsumerGeneratedNameIT {
     private static final Logger LOGGER = LoggerFactory.getLogger(ConsumerGeneratedNameIT.class);
-
+    static String kafkaUri;
     private static Map<String, Object> config = new HashMap<>();
     private static StrimziKafkaContainer kafkaContainer;
     private static final String BRIDGE_EXTERNAL_ENV = System.getenv().getOrDefault("EXTERNAL_BRIDGE", "FALSE");
     private static final String KAFKA_EXTERNAL_ENV = System.getenv().getOrDefault("EXTERNAL_KAFKA", "FALSE");
 
     static {
-        config.put(KafkaConfig.KAFKA_CONFIG_PREFIX + ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
+        if ("FALSE".equals(KAFKA_EXTERNAL_ENV)) {
+            kafkaContainer = new StrimziKafkaContainer();
+            kafkaContainer.start();
+            kafkaUri = kafkaContainer.getBootstrapServers();
+        } else {
+            // else use external kafka
+            kafkaUri = "localhost:9092";
+        }
+
+        config.put(KafkaConfig.KAFKA_CONFIG_PREFIX + ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaContainer.getBootstrapServers());
```

Comment:
Not sure about this. You first set the right address into the `kafkaUri` and then use the `kafkaContaiber.getBootstrapServers()` to configure the admin client?